modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,132 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 or later.
+
+=== Component versions included in {SMProductName} version 2.4
+
+|===
+|Component |Version
+
+|Istio
+|1.16
+
+|Envoy Proxy
+|1.22.9
+
+|Jaeger
+|1.42.0
+
+|Kiali
+|1.65
+|===
+
+=== Cluster-Wide Deployments
+This release introduces a generally available version of cluster-wide deployment. A cluster-wide deployment contains a Service Mesh Control Plane that monitors resources for an entire cluster. The control plane uses a single query across all namespaces to monitor each Istio or Kubernetes resource that affects the mesh configuration. Reducing the number of queries the control plane performs in a cluster-wide deployment improves performance.
+
+=== Discovery selectors Support
+This release introduces a generally available version of the `meshConfig.discoverySelectors` field, which can be used in cluster-wide deployment to limit which services the Service Mesh Control Plane can discover.
+
+[source,yaml]
+----
+spec:
+  meshConfig    
+    discoverySelectors:
+    - matchLabels:
+        env: prod
+        region: us-east1
+    - matchExpressions:
+      - key: app
+        operator: In
+        values:
+          - cassandra
+          - spark
+----
+
+=== Integration with cert-manager istio-csr
+Beginning with this release, {SMProductName} integrates with `cert-manager` and `istio-csr`. `cert-manager` adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. `cert-manager` provides and rotates an intermediate CA certificate for Istio. Integration with `istio-csr` enables users to delegate signing certificate requests from Istio to `cert-manager`.
```

Comment:
Could you also mention that SMCP v2.4 accepts CA certificate provided by cert-manager as `cacerts` secret?